tools/azure-devtools/src/azure_devtools/ci_tools/git_tools.py

```diff
@@ -90,8 +90,9 @@ def get_files_in_commit(git_folder, commit_id="HEAD"):
     return output.splitlines()
 
 def get_diff_file_list(git_folder):
-    """List of unstaged files.
+    """List of new files.
     """
     repo = Repo(str(git_folder))
-    output = repo.git.diff("--name-only")
+    repo.git.add("sdk")
```

Comment:
I really want to keep this file git generic, if you do a "git add" here you add side effect to a generic method, and this lib becomes unusable outside of the automation work

Reply:
I add new method and it does not influence old function now